Shokofin/API/Models/File.cs

```diff
@@ -80,7 +80,7 @@ public class Location
         /// where the <see cref="File"/> lies, with a leading slash applied at
         /// the start.
         /// </summary>
-        public string Path => "/" + RelativePath;
+        public string Path => System.IO.Path.DirectorySeparatorChar + RelativePath;
```

Comment:
We also need to make sure the path returned from  Shoko is OS agnostic/independent. It is currently dependent on the environment where Shoko Server runs (which may be Windows, Linux, or something else, etc.). Also, since we're doing path transformations we should probably cache the value for later re-use.

```suggestion
        public string Path =>
            __path != null ? (
                __path
             ) : (
                __path = System.IO.Path.DirectorySeparatorChar + RelativePath
                    .Replace('/', System.IO.Path.DirectorySeparatorChar)
                    .Replace('\\', System.IO.Path.DirectorySeparatorChar)
             );

        /// <summary>
        /// Cached path for later re-use.
        /// </summary>
        private string? __path { get; set; }
```